src/Facets/AmarokFacet.sol

```diff
@@ -25,25 +25,23 @@ contract AmarokFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
 
     /// Errors ///
```

Comment:
Maybe remove the headers if there is nothing underneath?

Reply:
ok